<a href="https://colab.research.google.com/github/dasom17/22_05_1st_Report/blob/main/report_1st.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

1. "환영합니다.", "파이썬의 세계에 오신 것을 환영합니다.", "파이썬은 강력합니다."를 화면에 출력하는 프로그램을 작성하시오. 스크립트 모드로 실행한다.

In [ ]:
print("환영합니다.", "파이썬의 세계에 오신 것을 환영합니다.", "파이썬은 강력합니다.")

환영합니다. 파이썬의 세계에 오신 것을 환영합니다. 파이썬은 강력합니다.


2. 다음 프로그램의 실행 결과를 쓰시오.

print("반갑습니다. 파이썬!")
print(2*3/10)
print("Hello", "World", "!!!")

In [ ]:
print("반갑습니다. 파이썬!")
print(2*3/10)
print("Hello", "World", "!!!")

반갑습니다. 파이썬!
0.6
Hello World !!!


1. 반갑습니다. 파이썬!
2. 0.6
3. Hello World !!!

3. 파이썬 쉘을 사용하여서 한 주가 몇 시간에 해당하는지를 계산해보자.

In [ ]:
print(7*24)

168


168시간이다.

4. 터틀 그래픽에서 거북이를 이동시켜 다음과 같은 그림을 그려보자. forward()와 right(), left() 함수만을 사용한다.

In [ ]:
import turtle
t = turtle.Turtle()
t.shape("turtle");
t.forward(100)
t.left(90)
t.forward(100)
t.right(90)
t.forward(100)
t.right(90)
t.forward(100)
t.left(90)
t.forward(100)
